### DISCUSSION TITLE
create a suite of test cases/data for validation

### DISCUSSION THREAD

**Discussion by globalflea** [serverlessworkflow/specification#525] [Ideas] [answered]
In order to ensure compliance with the specifications, it may be good to define a set of workflow definitions with json inputs and expected outputs for implementors to use to test for compliance - something like the examples provided but more comprehensive

Such a test suite will also allow the developer better understanding of what's expected by the specs

**Comment by cdavernas** [comment, marked as answer]
@globalflea It is an addition that I asked for the same reason some time ago. @tsurdilo didn't you guys release those?

**Comment by tsurdilo** [comment]
That's really important to have, agree. It falls under providing a TCK which has been a plan to do for some time. It is not a small task by any means and so far we haven't had anyone step up wanting to lead those efforts. 
Definitely will provide this before v1.0 release tho.
